examples/scenes/src/simple_text.rs

```diff
@@ -161,7 +162,13 @@ impl SimpleText {
             .font_size(size)
             .transform(transform)
             .glyph_transform(glyph_transform)
-            .normalized_coords(var_loc.coords())
+            .normalized_coords(
+                var_loc
+                    .coords()
+                    .iter()
+                    .copied()
+                    .map(NormalizedCoord::to_bits),
```

Comment:
I think:
```suggestion
                    .map(|coord| coord.to_bits()),
```
should work and is slightly nicer

Reply:
Looking again, I don't think this is necessary.